pkgs/development/python-modules/xvfbwrapper/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, xorgserver
+}:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
```

Comment:
no `name`

Reply:
Not sure what you mean sorry. There should be no name attr? https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.md recommends this